dbt_subprojects/dex/models/trades/base/_schema.yml

```diff
@@ -996,3 +996,20 @@ models:
             - evt_index
       - check_dex_base_trades_seed:
           seed_file: ref('xchange_base_base_trades_seed')
+
+  - name: infusion_base_base_trades
+    meta:
+      blockchain: base
+      sector: dex
+      project: infusion
```

Comment:
Please add `infusion` in `dex.info`:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/dex/models/dex_info.sql